app/courses/page.tsx

```diff
@@ -57,34 +70,38 @@ const Path = ({
   return (
     <Link href={isFree ? `/courses/${slug}` : `/signup`}>
       <Card
+        variant={isFree ? "whiteShadow" : "locked"}
```

Comment:
Beautiful

Reply:
Only thing I might say is `locked` isn't really something that makes sense for a `Card` component, outside the context of this page, when a dev comes a long and sees "oh the card component has a locked component" what are they supposed to think?

That being said, it's fine, just thought I'd let you know.